src/dashboard/NewSideBar.jsx

```diff
@@ -0,0 +1,140 @@
+import { TbShoe } from "react-icons/tb";
+import { CiVault } from "react-icons/ci";
+import { Link } from 'react-router-dom';
+
+const sections = [
+    { title: 'My Society', path: '/my-society', Icon: TbShoe },
+    { title: 'Discover', path: '/discover', Icon: ExploreIcon },
+    { title: 'Groups', path: '/groups', Icon: GroupsIcon },
+    { title: 'The Vault', path: '/vault', Icon: CiVault },
+    { title: 'Messages', path: '/messages', Icon: MessageIcon },
+    // Additional sections can be added here...
+];
+
+const NewSidebar = ({ sidebarOpen, toggleSidebar }) => {
+    const isMobile = useMediaQuery('(max-width:600px)');
+    const iconColor = isMobile ? 'black' : 'black'; // Ensures icons are black on desktop
+    const backgroundColor = isMobile ? 'white' : 'white'; // Ensures background is white on desktop
+
+    return (
+        <>
+            {isMobile && (
+                <IconButton
+                    onClick={toggleSidebar}
+                    color="inherit"
+                    sx={{
+                        position: 'fixed',
+                        top: 8,
+                        right: 8,
+                        zIndex: 1300,
+                        display: 'block',
+                        color: sidebarOpen ? 'black' : 'white', // Adjust menu button color based on sidebar state
+                    }}
+                >
+                    <MenuIcon />
+                </IconButton>
+            )}
+            <Drawer
+                variant={isMobile ? "temporary" : "permanent"}
+                anchor="left"
+                open={sidebarOpen || !isMobile}
+                onClose={toggleSidebar}
+                PaperProps={{
+                    sx: {
+                        width: isMobile ? '100vw' : 240,
+                        height: '100vh',
+                        boxSizing: 'border-box',
+                        display: 'flex',
+                        flexDirection: 'column',
+                        position: 'fixed',
+                        zIndex: 1200,
+                        backgroundColor: backgroundColor, // Apply dynamic background color
+                        color: iconColor, // Text color inherited by children
+                    },
+                }}
+            >
+                <Box
+                    display="flex"
+                    justifyContent="center"
+                    alignItems="center"
+                    p={2}
+                    sx={{
+                        background: 'black', // Adjust if you want the circle background to be different
+                        borderRadius: '50%',
+                        width: 200,
+                        height: 200,
+                        marginTop: '96px',
+                        alignSelf: 'center',
+                    }}
+                >
+                    <img
+                        alt="Sneaker Society Logo"
+                        src={`../../assets/SNEAKER SOCIETY (Transparency).png`}
```

Comment:
Make this an import from the top and then use the variable as you have done in the past. On deployment to PROD it causes an error if imported this way.